music21/musicxml/xmlToM21.py

```diff
@@ -4026,16 +4026,19 @@ def updateLyricsFromList(self, n, lyricList):
             if lyricObj.number == 0:
                 lyricObj.number = currentLyricNumber
             # If there is more than one text (and, therefore, syllabic), create two lyric objets
-            if lyricObj.text is not None and isinstance(lyricObj.text, list):
-                text_list = lyricObj.text
-                syllabic_list = lyricObj.syllabic
-                for i, t in enumerate(text_list):
-                    uniqueLyricObj = copy.copy(lyricObj)
-                    uniqueLyricObj.text = t
-                    if syllabic_list and len(syllabic_list) > i:
-                        uniqueLyricObj.syllabic = syllabic_list[i]
-                    n.lyrics.append(uniqueLyricObj)
-                    currentLyricNumber += 1
+            if lyricObj.text is not None:
+                if isinstance(lyricObj.text, list):
+                    text_list = lyricObj.text
+                    syllabic_list = lyricObj.syllabic
+                    for i, t in enumerate(text_list):
+                        uniqueLyricObj = copy.copy(lyricObj)
+                        uniqueLyricObj.text = t
+                        if syllabic_list and len(syllabic_list) > i:
+                            uniqueLyricObj.syllabic = syllabic_list[i]
+                        n.lyrics.append(uniqueLyricObj)
```

Comment:
If it is a list: parse it lyric by lyric. Otherwhise: introduce the lyric in the object.